pages/search/search.jsx

```diff
@@ -38,8 +40,10 @@ class Search extends React.Component {
   updateBrowserHistory() {
     let keywordSearched = this.state.keywordSearched;
     let location = { pathname: this.props.location.pathname };
-    let query = {};
-
+    let query = {
```

Comment:
Let's add `helpType` to `query` only if `helpType` exists

Reply:
Similar to Line 47? ⬇️

Reply:
yup yup